joblib/_dask.py

```diff
@@ -121,10 +121,13 @@ def __init__(self, tasks):
 
     def __call__(self, tasks=None):
         results = []
-        with parallel_backend('dask'):
-            for func, args, kwargs in tasks:
-                results.append(func(*args, **kwargs))
-        return results
+        try:
+            with parallel_backend('dask'):
+                for func, args, kwargs in tasks:
+                    results.append(func(*args, **kwargs))
+            return dict(status="Done", result=results)
+        except BaseException as e:
+            return dict(status="Error", result=e)
```

Comment:
These status should be constants from joblib to avoid confusion in the maintenance.